extensions/amp-access/0.1/amp-access.js

```diff
@@ -777,24 +567,11 @@ export class AccessService {
    * @private
    */
   reportViewToServer_() {
-    return this.adapter_.pingback().then(() => {
-      dev().fine(TAG, 'Pingback complete');
-      this.analyticsEvent_('access-pingback-sent');
-    }).catch(error => {
-      this.analyticsEvent_('access-pingback-failed');
-      throw user().createError('Pingback failed: ', error);
-    });
-  }
-
-  /**
-   * @param {string} className
-   * @param {boolean} on
-   * @private
-   */
-  toggleTopClass_(className, on) {
-    this.vsync_.mutate(() => {
-      this.getRootElement_().classList.toggle(className, on);
-    });
+    const promises = [];
+    for (let i = 0; i < this.sources_.length; i++) {
+      promises.push(this.sources_[i].reportViewToServer());
```

Comment:
Do we need to check for `pingbackEnabled`?

Reply:
good point. related, the existing code would never send the access-viewed analytics event unless pingback was enabled, was that desired?

Reply:
Probably not too great. So disabled pingback is relatively a rare thing and there are other "pageview" related analytics events. So it probably just never came up.

Reply:
got it